.github/workflows/artifact-tests.yml

```diff
@@ -1,13 +1,21 @@
 name: artifact-unit-tests
-on: push
+on:
```

Comment:
Also updating these to match our filters in [unit-tests.yml](https://github.com/actions/toolkit/blob/master/.github/workflows/unit-tests.yml)